core/domain/exp_domain_test.py

```diff
@@ -1724,6 +2775,21 @@ def test_validation(self) -> None:
         }
         exploration.validate()
 
+        # Interaction validations.
+        new_exploration = exp_domain.Exploration.create_default_exploration(
+            'test_id')
+        state = new_exploration.states['Introduction']
+
+        self._tests_continue_interaction(state, new_exploration)
+        self._tests_numeric_interaction(state, new_exploration)
+        self._tests_fraction_interaction(state, new_exploration)
+        self._tests_number_with_units_interaction(state, new_exploration)
+        self._tests_multiple_choice_interaction(state, new_exploration)
+        self._tests_item_selection_choice_interaction(state, new_exploration)
+        self._tests_drag_and_drop_interaction(state, new_exploration)
+        self._tests_text_interaction(state, new_exploration)
+        self._tests_end_interaction(state, new_exploration)
```

Comment:
Why is this done this way? And not by directly doing `test_continue_interaction`? You can prepare the content in `setUp` if needed.

Reply:
Oh, as this was part of the `validate` function I did it like this. If you want I can do it that way.
Thanks